app/controllers/planning_applications_controller.rb

```diff
@@ -177,6 +168,16 @@ def validate_documents
 
 private
 
+  def audit(activity_type, audit_comment = nil, *activity_information)
```

Comment:
I think it's OK to move this method up into application_controller, so it's available to both planning applications and documents controller, and you don't have the duplicate the code